landoapi/transplant_client.py

```diff
@@ -48,8 +52,29 @@ def land(self, ldap_username, hgpatch, tree, pingback, request):
             }
         )
 
-        # Transplant API is responding with a created request_id of the job
-        return result.get('request_id') if result else None
+        if result:
+            logger.info(
+                {
+                    'service': 'transplant',
+                    'username': ldap_username,
+                    'msg': 'patch sent to transplant service',
+                }, 'transplant.success'
+            )
+            return result.get('request_id')
+        else:
+            # Transplant API responded with no data, indicating an error of
+            # some sort.
+            logger.info(
+                {
+                    'service':
+                    'transplant',
+                    'username':
+                    ldap_username,
+                    'msg':
+                    'received an empty response from the transplant service',
+                }, 'transplant.failure'
```

Comment:
at the end of this line you can add a comment like: `# yapf: disable` to prevent the linter from formating this hash. I don't think this is good style just because the linter says so, it's clearly a bug and we know better :)